### PR TITLE
Include CVEs (and similar ids) at the bottom of the GitHub issue body

### PR DESCRIPTION
**Pull request by watson**

### CODE REVIEW

File: cli/utils.js [hunk 1]
```diff
@@ -120,6 +120,17 @@ const getGraph = (issue, prefix, showFullManifest) => {
         .join('\r\n');
 };
 
+const getIdentifiers = (issue) => {
+    let s = '';
+    for (const ids of Object.values(issue.issueData.identifiers)) {
+        if (ids.length === 0) continue;
```
Comment (azasypkin) [inline cli/utils.js]: __optional nit:__ it looks like this `if` isn't strictly necessary - the next `for` will basically be a no-op. Or maybe just ``Object.values(issue.issueData.identifiers).flat().map((id) => `- ${id}\r\n`).join('')`` would work?

File: cli/batch.js [hunk 2]
```diff
@@ -122,7 +123,7 @@ ${getGraph(issue, '* ', showFullManifest)}
 
 ${issue.issueData.description}
 - [${issue.id}](${issue.issueData.url})
-</details>`
+${getIdentifiers(issue)}</details>`
```
Comment (azasypkin) [inline cli/batch.js]: I noticed that the content rendered inside of `<details>` isn't searchable (e.g. I cannot find https://github.com/elastic/security/issues/482 by `"CVE-2021-3749"`), do you think we can move identifiers outside of `<details>` or make them searchable in any other way?

**Review by azasypkin — APPROVED** [review]
Looks great, thanks! Just left one optional nit and question about "search-ability" of the identifiers.

File: cli/utils.js (hunk 1 above)
Comment (watson) [reply cli/utils.js]: Ah you're right. I had some more core in that loop previously that I wanted to skip in this case, but now that it's removed I can also just remove the `if`

File: cli/batch.js (hunk 2 above)
Comment (watson) [reply cli/batch.js]: Oh! That's a bummer. I didn't even think of that. I'll find a good workaround...

Comment (watson) [reply cli/batch.js]: Option no 1 - Include IDs in header:

image

Option no 2 - Comma separated list of IDs at the bottom of body:

image

Comment (watson) [reply cli/batch.js]: I've updated the PR to implement option number one (just as an example, we can change it to something else if you like).

As you might have noticed, we only use the `<details>` block when in batch mode. So, I've kept the IDs at the bottom of the body when not in batch mode and included them in the title when in batch mode.

Comment (watson) [reply cli/batch.js]: Btw, I've verified that anything inside of the `<summary>` block is searchable even though it's technically inside the `<details>` block as well.

File: cli/utils.js (hunk 1 above)
Comment (watson) [reply cli/utils.js]: Fixed

File: cli/batch.js (hunk 2 above)
Comment (azasypkin) [reply cli/batch.js]: > I've updated the PR to implement option number one (just as an example, we can change it to something else if you like).

Sounds good to me, thanks for the change!

> Btw, I've verified that anything inside of the summary block is searchable even though it's technically inside the details block as well.

Ah, good to know

**Pull request merged**